packages/lib-sourcify/test/functions.spec.ts

```diff
@@ -44,10 +47,41 @@ describe('Verify Solidity Compiler', () => {
           },
         },
       });
-    } catch (e) {
-      //
+    } catch (e: any) {
+      expect(e.message.startsWith('Compiler error:')).to.be.true;
+    }
+  });
+  it('Should compile with solcjs', async () => {
+    const realPlatform = process.platform;
+    Object.defineProperty(process, 'platform', {
+      value: 'not existing platform',
+      writable: false,
+    });
+    try {
+      const compiledJSON = await useCompiler('0.8.9+commit.e5eed63a', {
+        language: 'Solidity',
+        sources: {
+          'test.sol': {
+            content: 'contract C { function f() public  {} }',
+          },
+        },
+        settings: {
+          outputSelection: {
+            '*': {
+              '*': ['*'],
+            },
+          },
+        },
+      });
+      expect(compiledJSON?.contracts?.['test.sol']?.C).to.not.equals(undefined);
+    } catch (e: any) {
+      expect.fail('Compiler error:');
```

Comment:
Do we expect it to fail? Doesn't it pass the tests with this?

Reply:
typo again

Reply:
no sorry, with `expect.fail()` you are forcing the test to fail, and it's right

Reply:
https://unexpected.js.org/api/fail/

Reply:
Ok let's add the error message there then. Because if it will be caught there won't be a way to see why it failed.

Reply:
ok